images/devtools-golang-v1beta1/context/Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM docker.io/safewaters/docker-lock:latest@sha256:432d90ddc2891f4845241adc63e5eef2dd1486fa14ea7882433cbd3f8ed64622 AS docker-lock
 FROM golangci/golangci-lint:v1.48.0-alpine@sha256:e9ce901b7974d641d6288b070fc619d199a823d0f696563fec8be90f19789b57 AS golangci-lint
 FROM docker.io/mikefarah/yq:4@sha256:856f7ab12c58608d422a742a4917e6998ad065d9d48a5b1cd69c3ce8fa80f3fa AS yq
-
-
-# golangci-lint does not work with go1.18
-# https://github.com/golangci/golangci-lint/issues/2649
+FROM docker.io/hadolint/hadolint:v2.10.0@sha256:93f0afd12c3be5d732227c0226dd8e7bb84f79319a773d7f8519e55f958ba415 AS hadolint
+FROM docker.io/goodwithtech/dockle:v0.4.6@sha256:534ecfe2204403e1b563d61b255ddd4762b20c9d72ce7af239fb1135be6c9569 AS dockle
+FROM docker.io/moby/buildkit:v0.10.3-rootless@sha256:22e39442689ddbfd938892b33a12cc9bc76c7e71e07cbaa20c8272a7aab2fd59 AS buildkit
+FROM gcr.io/go-containerregistry/crane:v0.11.0@sha256:40823185f184cfc2fed19318988ad5119f4d2fd1eadee51d8b2b365016a6c03b AS crane
 FROM golang:1.18.5-bullseye@sha256:ac2ea4e233e5f1bb4407893f58d4e33d908945d4163e5848160ab588d1d6a456 AS base
```

Comment:
@aucampia fully qualified docker image reference?

https://playbook.internal.coop/guidelines/security/docker-images-and-containers.html?h=docker#use-fully-qualified-docker-image-references

Reply:
fixed